### PR TITLE
Minor cleanup: leftover file removal and directory structure cleanup

### PR DESCRIPTION
**Pull request by neteler**
- delete `.bak` files
- move `.ico` into `casas/icons/`
- set executable flag for Perl files missing it

### CODE REVIEW

File: casas_gis_old/PerlScripts/SubtractOutput.pl
Comment (luisponti) [inline casas_gis_old/PerlScripts/SubtractOutput.pl:1]: Is there a non-empty version somewhere else?

File: casas_gis_old/PerlScripts/getBoxplotColorRule.pl
Comment (luisponti) [inline casas_gis_old/PerlScripts/getBoxplotColorRule.pl:1]: Is there a non-empty version somewhere else?

File: casas_gis_old/PerlScripts/subtract.pl
Comment (luisponti) [inline casas_gis_old/PerlScripts/subtract.pl:1]: Is there a non-empty version somewhere else?

File: casas_gis_old/PerlScripts/test.pl
Comment (luisponti) [inline casas_gis_old/PerlScripts/test.pl:1]: Forgot what this was about

File: casas_gis_old/casas/scripts/usa [hunk 1]
```diff
@@ -40,7 +40,7 @@
 #%Module
 #%  description: Map the output of CASAS models for the conterminous United States, Mexico, and Central America
 #%  keywords: models
-#%End 
+#%End
```
Comment (luisponti) [inline casas_gis_old/casas/scripts/usa:43]: Not sure what is being done here: removed and then added the same line?

File: casas_gis_old/scripts/SubtractOutput.pl
Comment (luisponti) [inline casas_gis_old/scripts/SubtractOutput.pl:1]: Is there a non-empty version somewhere else?

**Review by luisponti — APPROVED** [review]
Thanks, @neteler 
See my comments if useful.

File: casas_gis_old/PerlScripts/SubtractOutput.pl
Comment (neteler) [reply casas_gis_old/PerlScripts/SubtractOutput.pl:1]: Note that this is not empty, but just changed the file permissions.

File: casas_gis_old/PerlScripts/getBoxplotColorRule.pl
Comment (neteler) [reply casas_gis_old/PerlScripts/getBoxplotColorRule.pl:1]: (see above)

File: casas_gis_old/PerlScripts/subtract.pl
Comment (neteler) [reply casas_gis_old/PerlScripts/subtract.pl:1]: (see above)

File: casas_gis_old/PerlScripts/test.pl
Comment (neteler) [reply casas_gis_old/PerlScripts/test.pl:1]: (see above)

File: casas_gis_old/scripts/SubtractOutput.pl
Comment (neteler) [reply casas_gis_old/scripts/SubtractOutput.pl:1]: (see above)

File: casas_gis_old/casas/scripts/usa (hunk 1 above)
Comment (neteler) [reply casas_gis_old/casas/scripts/usa:43]: Just an empty space after the `d` of `End` was deleted.

**Comment by luisponti** [comment]
Ok, resolved all conversations. Thanks, @neteler

**Pull request merged**